addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -911,3 +911,9 @@ msgstr ""
 msgctxt "#31611"
 msgid "System"
 msgstr ""
+#: /xml/SkinSettings.xml
+#. Label to show TV?Radio menu action
+msgctxt "#31612"
+msgid "Choose PVR action on menu selection"
```

Comment:
Can we find something more self explaining? @da-anda do you have an idea? How do we call the items on the left? "Home screen menu item"? Then, this could read "PVR home screen menu item select action"?

Reply:
It's hard to come up with a short label that describes this. So ideally, if we would extend this feature to all/more home screen menu items (does this make sense?), we could either have a dedicated settings section for it, or at least some headline with a short description and below that then a list of all the menu items.

That said, could we move this setting to where users can already manipulate the home screen menu settings? So right next to the show/hide toggle? Then we could use a label like: `menu  item click action`. But this is ofc a lot more work than this PR signed up for.

If we stick to the scope of what this PR tries to achieve, then something like this maybe `Home screen menu action for "live TV"`?